tests/components/ring/test_init.py

```diff
@@ -1,67 +1,75 @@
 """The tests for the Ring component."""
 
-from datetime import timedelta
-from unittest.mock import patch
-
+from freezegun.api import FrozenDateTimeFactory
 import pytest
-import requests_mock
 from ring_doorbell import AuthenticationError, RingError, RingTimeout
 
 from homeassistant.components import ring
 from homeassistant.components.camera import DOMAIN as CAMERA_DOMAIN
 from homeassistant.components.light import DOMAIN as LIGHT_DOMAIN
 from homeassistant.components.ring import DOMAIN
+from homeassistant.components.ring.const import SCAN_INTERVAL
 from homeassistant.config_entries import SOURCE_REAUTH, ConfigEntryState
 from homeassistant.const import CONF_USERNAME
 from homeassistant.core import HomeAssistant
 from homeassistant.helpers import entity_registry as er, issue_registry as ir
 from homeassistant.setup import async_setup_component
-from homeassistant.util import dt as dt_util
 
-from tests.common import MockConfigEntry, async_fire_time_changed, load_fixture
+from tests.common import MockConfigEntry, async_fire_time_changed
 
 
-async def test_setup(hass: HomeAssistant, requests_mock: requests_mock.Mocker) -> None:
+async def test_setup(hass: HomeAssistant, mock_ring_client) -> None:
     """Test the setup."""
     await async_setup_component(hass, ring.DOMAIN, {})
 
-    requests_mock.post(
-        "https://oauth.ring.com/oauth/token", text=load_fixture("oauth.json", "ring")
-    )
-    requests_mock.post(
-        "https://api.ring.com/clients_api/session",
-        text=load_fixture("session.json", "ring"),
-    )
-    requests_mock.get(
-        "https://api.ring.com/clients_api/ring_devices",
-        text=load_fixture("devices.json", "ring"),
-    )
-    requests_mock.get(
-        "https://api.ring.com/clients_api/chimes/999999/health",
-        text=load_fixture("chime_health_attrs.json", "ring"),
-    )
-    requests_mock.get(
-        "https://api.ring.com/clients_api/doorbots/987652/health",
-        text=load_fixture("doorboot_health_attrs.json", "ring"),
-    )
+
+async def test_setup_entry(
+    hass: HomeAssistant,
+    mock_ring_client,
+    mock_config_entry: MockConfigEntry,
+) -> None:
+    """Test setup entry."""
+    mock_config_entry.add_to_hass(hass)
+    assert await hass.config_entries.async_setup(mock_config_entry.entry_id)
+    await hass.async_block_till_done()
```

Comment:
Didn't we have a fixture for this

Reply:
Switched this to use the fixture and did the same for the test below this one `test_setup_entry_device_update`